pkgs/development/python-modules/color-operations/default.nix

```diff
@@ -1,14 +1,14 @@
 {
   lib,
   buildPythonPackage,
-  fetchFromGitHub,
-  pytestCheckHook,
   pythonOlder,
-
-  colormath,
+  fetchFromGitHub,
   cython,
-  oldest-supported-numpy,
   setuptools,
+  oldest-supported-numpy,
+  colormath,
+  pytestCheckHook,
+  stdenv,
```

Comment:
Why reorder these? Especially away from a grouped + sorted order? The new order looks arbitrary.

Reply:
The order is the one recommended in the manual, i.e. the order of apparition in the derivation.
I added comments to highlight the categories.